src/utils/copyText.ts

```diff
@@ -0,0 +1,7 @@
+export function copyText(text: string) {
+    if (navigator.clipboard.writeText) {
```

Comment:
`navigator?.clipboard?.writeText` ?

Reply:
I think `if` is better for possible stackatrace investigation

Reply:
but it is possible that application will fail, if there's no `clipboard` in `navigator` proto. We don't use `try ... catch` when using `copyText`.
I suggested not removing `if`, but add checks like `if (navigator?.clipboard?.writeText) { ... }`